src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -403,6 +446,25 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
     }
   }
 
+  /**
+   * Displays dialog to prompt user whether to change the Completing Party.
+   * @returns a promise that is fulfilled/rejected when the user responds
+   */
+  private async confirmReassignCp (): Promise<any> {
+    // open confirmation dialog and wait for response
+    return this.$refs.changeCpDialog.open(
+      'Change Completing Party?',
+      this.changeCpMessage,
+      {
+        width: '45rem',
+        persistent: true,
+        yes: 'Restore original Completing Party',
+        no: 'Cancel',
+        cancel: null
```

Comment:
Note the usage here...
- Yes and No both resolve the promise, while Cancel rejects it.
- I display both Yes and No buttons (with overridden text).
- I disable the Cancel button.
- Therefore, I only need to check the "then()" for this promise (and ignore the "catch()").